### DISCUSSION TITLE
Can I serve shared files which are HTML as web pages whcih open in a browser.

### DISCUSSION THREAD

**Discussion by empedia** [drakkan/sftpgo#1595] [General]
Hi,

I have a set of HTML files which I want to share and have them rendered in the web browser. 

I have uploaded a file with a .html extension and created a share for it and generated a link to the file to be able to access it uncompressed. The HTML is completely self-contained and does not require any additional files to be used.

What I want to do is have the page open as  web page in the browser but what actually happens is that I get prompted to download the file. I am assuming this is because the MIME type is not being provided as text/html - is there any way of achieving this? 

I can't use WebDAV for this I believe because I just want a basic URL which I can share with people.

Many thanks in advance.

**Comment by drakkan** [comment]
This is not supported sorry, SFTPGo does not render HTML pages, you have to use a webserver for this
